### PR TITLE
ui: improved ui for note editing \n Clickable area for editing notes …

### PR DESCRIPTION
**Pull request by jayantpranjal0**
Clickable area for editing notes or creating new notes was only on the text area. Modified it to include the text box.
Fixes: #176

### CODE REVIEW

File: app/components/EditableLabel/EditableLabel.js [hunk 1]
```diff
@@ -146,10 +146,11 @@ export default class EditableLabel extends React.Component {
       ? this.state.text
       : this.props.labelPlaceHolder || DEFAULT_LABEL_PLACEHOLDER;
     return (
-      <div className={this.props.containerClassName}>
+      <div className={this.props.containerClassName}
+        onClick={this.handleFocus}
+      >
```
Comment (Abhijay007) [inline app/components/EditableLabel/EditableLabel.js:151]:
```suggestion
     <div className={this.props.containerClassName} onClick={this.handleFocus}>
```

Looks good to me, but could you please format it properly?

**Review by Abhijay007 — COMMENTED** [review]
Formatting required.

**Review by lrasmus — APPROVED** [review]
Change looks good, thanks!

**Comment by lrasmus** [comment]
Thank you for this helpful fix @jayantpranjal0, and thank you @Abhijay007 for the review!

**Pull request merged**